modules/jt-extending-openshift-pipelines-capabilities-using-custom-tasks-and-scripts.adoc

```diff
@@ -1,11 +1,11 @@
 // Module included in the following assembly:
 //
-// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+// jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc
 
-[id="jt-extending-tekton-capabilities-using-custom-tasks-and-scripts_{context}"]
-= Extending Tekton capabilities using custom tasks and scripts
+[id="jt-extending-openshift-pipelines-capabilities-using-custom-tasks-and-scripts_{context}"]
+= Extending OpenShift Pipelines capabilities using custom tasks and scripts
 
-In Tekton, if you do not find the right task in Tekton Hub, or need greater control over tasks, you can create custom tasks and scripts to extend Tekton's capabilities.
+In OpenShift Pipelines, if you do not find the right task in Tekton Hub, or need greater control over tasks, you can create custom tasks and scripts to extend OpenShift Pipelines's capabilities.
```

Comment:
```suggestion
In OpenShift Pipelines, if you do not find the right task in Tekton Hub, or need greater control over tasks, you can create custom tasks and scripts to extend the capabilities of OpenShift Pipelines.
```
Per IBM SG: `Do not use 's to show the possessive form of abbreviations or product names.`

Reply:
Updated. Thanks.